src/docs/product/releases/index.mdx

```diff
@@ -352,3 +352,25 @@ You can also use our [API](/api/releases/create-a-new-deploy-for-an-organization
 JavaScript and React Native projects can utilize release artifacts to unminify or symbolicate error stack traces, you can find the artifacts uploaded to Sentry by navigating to **[Project] » Project Settings » Source Maps**.
 
 To learn more, please check out our [Source Maps](/platforms/javascript/sourcemaps/) docs.
+
+## Archiving Releases
+
+If you don't wish to see a release anymore you can archive it either using CLI:
```

Comment:
This is a great start, but can we say a bit more about the why a customer might benefit from archiving a release? Similar to https://sentry-docs-git-feat-release-archiving.sentry.dev/product/sentry-basics/environments/#hidden-environments?